README.md

```diff
@@ -24,33 +24,14 @@ render correctly as HTML in local preview.
 
 ### Broken links checker
 
-```shell
-cd utils/blc
-docker build -t blc .
-docker run --rm --network host blc blc http://0.0.0.0:8000 -r | tee blc.txt
-```
-
-Above will check all of the links in the whole documentation recursively and
-generate the `blc.txt`, which will provide information on which links are
-reachable, and which are not.
-
-Checking the whole documentation can be time-consuming task (around 15 minutes).
-
-It is a good practice to verify at least the document you have edited, before
-publishing them. You can check a single page for broken links as well, simply
-by providing it's url. For example:
+Should be installed through `requirements.txt`. Following command output
+`linkchecker-out.txt`, feel free to adjust other parameters according to your
```

Comment:
I do not understand this sentence. Also, I do not see the `linkchecker-out.txt` being used

Reply:
@macpijan can you propose change?

I'm not sure how to explain that differently at this point:
* linkchecker is installed through `pip install -r requirements.txt` so if one use venv, which one should to preview locally, then linkchecker is there and can be run to test for broken links
* If one use command that I suggest one would get `linkchecker-out.txt` file in which it is clear which links are broken not much more to understand here or explain usage.

If your expectations would be different for that description please let me know what is missing and will try to explain that better.

Reply:
> Should be installed through `requirements.txt`

This sentence lacks subject. What should be installed through requirements?

```
To check for broken links we use `linkchecker`.  It should be installed through [requirements.txt](link_to_file?).
```

```
Following command output `linkchecker-out.txt`
```

This may sound confusing because it is incorrect in the grammar sense. Either:

```
Following command produces `linkchecker-out.txt` file, which contains (describe here what it contains?)
```

OR

```
Following command outputs `linkchecker-out.txt` file, which contains (describe here what it contains?)
```

The finish with the sentence `Feel free to adjust other parameters according to your needs`

Reply:
Fixed: https://github.com/Dasharo/docs/pull/512/commits/6b5af9801e18eeb7b255900e18eb6c8b3b18551c

Reply:
Yes exactly @miczyg1 got my point. I think it is fixed now.